x-pack/plugins/cloud/public/plugin.tsx

```diff
@@ -62,6 +64,10 @@ export class CloudPlugin implements Plugin<CloudSetup> {
       decodedId = decodeCloudId(id, this.logger);
     }
 
+    core.getStartServices().then(([coreStart]) => {
+      coreStart.chrome.setIsCloudEnabled(this.isCloudEnabled);
```

Comment:
Unless I'm missing something, `chrome` [start contract](Unless I'm missing something) doesn't have a `setIsCloudEnabled` method. CI's reporting as much too: type not decared.

Reply:
That's what i am adding in this PR 😊 (https://github.com/elastic/kibana/pull/160535/files#diff-327964d34602d7b92f6811379eb9ed6358489b433d2fdf1cd243e5e1b741dd31).
Will check CI